enterprise/server/remote_execution/execution_server/execution_server.go

```diff
@@ -53,30 +52,30 @@ var (
 	enableActionMerging               = flag.Bool("remote_execution.enable_action_merging", true, "If enabled, identical actions being executed concurrently are merged into a single execution.")
 )
 
-func fillExecutionFromSummary(summary *espb.ExecutionSummary, execution *tables.Execution) {
+func fillExecutionFromActionMetadata(md *repb.ExecutedActionMetadata, execution *tables.Execution) {
```

Comment:
we might want to keep around the old method for backwards compatibility, in case customers are running old executor versions that are still setting ExecutionSummary. These are just used for timing data in the UI though, currently, so I don't feel super strongly about it. @tylerwilliams wdyt?

Reply:
+1 to making this backwards compatible even though we expect it to be short lived

Reply:
Done!